### ISSUE TITLE
VSCodium shows unchanged and changed files in git section

### ISSUE THREAD

**Issue by Neiz-Kap** [VSCodium/vscodium#1712]
**Describe the bug**
Hello! 
Recently I changed my OS (from Windows to ArcoLinux) and saved my projects with .git folder and my new features in some files to flash drive. Also I exported data from VSCode and applyed them for VSCodium, when I installed VSCodium

And when I open any project, **I can see definitely all files in git section: changed and unchanged**.

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/docs/index.md) and [the Troubleshooting page](https://github.com/VSCodium/vscodium/blob/master/docs/troubleshooting.md) and my issue is not mentioned there.

**To Reproduce**
I'm not sure it could be reproduced  

**Expected behavior**
There should be only modified files

**Screenshots**

![image](https://github.com/VSCodium/vscodium/assets/119663114/bfbfb04a-975d-4295-b2fd-1f48c1bf8845) 
**Desktop (please complete the following information):**

 - OS: ArcoLinux, Gnome45.1,
 - Architecture [e.g. x64, ia32, arm64]
 - Version 1.84.0
 - App Manager: pacman
 - Sandboxed: I installed the VSCodium from [arcolinux_repo_xlarge](https://gitlab.com/arcolinux/arcolinux_repo_xlarge/-/blob/main/x86_64/vscodium-bin-1.84.0.23306-1-x86_64.pkg.tar.zst?ref_type=heads)

**Additional context**
Release: 23306
Commit: 695a9146b1347633e608f9f166c81b5b2843b492
Date: 2023-11-02T15:23:23.641Z
Electron: 25.9.2
ElectronBuildId: undefined
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Linux x64 6.5.9-arch2-1

- I guess, It can be useful note for solving , **I can "discard changes" for unedited files and the file disappears from the list.** 
- When I used git stash, all files were stashed. After I typed `git stash pop` and they all stay
- git pull/fetch doesn't help

**Comment by Neiz-Kap** [comment]
Albeit I tried to open the project in Code-OSS and I have the same case. 
I copied initial code (from left) to current and it stays as there was. Here I checked if there is symbol issue, I don't know

**Comment by daiyam** [comment]
Have you tried `git status`? *Silent* diffs can be due to some spaces or a line break.

https://stackoverflow.com/questions/19467836/how-to-ignore-files-from-git-status-if-only-white-spaces-and-line-breaks-chang

**Comment by Neiz-Kap** [comment]
Hey, sorry for the late reply! Indeed the issue was due to difference in line endings between Windows and Linux distributions.

I remember to save the command, that helped me, but not totally. 
`git diff -w | git apply --cached --ignore-whitespace`
New files are not considered by this command. Note this

Here's the information, that could explain a reason of that and resolve the issue: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#refreshing-a-repository-after-changing-line-endings

The issue page can be closed, because the problem is on git and encoding side, than VSCodium
Many thanks for your work, I've been using VSCodium since I got back to Linux (since November 2023)
